src/main/java/io/jenkins/plugins/services/impl/HttpClientJiraIssues.java

```diff
@@ -0,0 +1,100 @@
+package io.jenkins.plugins.services.impl;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Strings;
+import io.jenkins.plugins.models.JiraIssue;
+import io.jenkins.plugins.models.JiraIssues;
+import io.jenkins.plugins.services.ConfigurationService;
+import org.apache.http.Header;
+import org.apache.http.client.config.RequestConfig;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.net.URLEncoder;
+import java.util.Collections;
+import java.util.List;
+
+public class HttpClientJiraIssues extends HttpClient {
+  private final ConfigurationService configurationService;
+  private Logger logger = LoggerFactory.getLogger(HttpClientJiraIssues.class);
+
+  private ObjectMapper mapper = new ObjectMapper();
```

Comment:
this can be initialised only once:

```suggestion
  private static final ObjectMapper MAPPER = new ObjectMapper();
```

Reply:
this one was apparently never even used